test/extended/networking/tuning.go

```diff
@@ -43,6 +44,53 @@ var whitelistedSysctls = []SysctlVariant{
 	// {Sysctl: "net.ipv6.neigh.IFNAME.retrans_time_ms", Value: "1010", Path: "/proc/sys/net/ipv6/neigh/net1/retrans_time_ms"},
 }
 
+// getLowestIfindexInterfaceFromUeventData parses the output of one or more concatenated 'uevent' files
+// from /sys/class/net/*/uevent and returns the name of the interface with the lowest ifindex
+func getLowestIfindexInterfaceFromUeventData(data string) (string, error) {
```

Comment:
This seems really complicated. Any reason to not just use `ip link`?

Also, the lowest-ifindex interface is probably `lo`, which the test was trying to avoid.

`ip -o link show | grep -v LOOPBACK | head -1 | sed -e 's/:.*//'`

Reply:
Yeah, that was my next step. Wasn't sure if /sbin/ip was even in the container image.